spyder/plugins/explorer/widgets/explorer.py

```diff
@@ -997,6 +997,9 @@ def get_selected_filenames(self):
         else:
             fnames = [self.get_filename(self.currentIndex())]
 
+        if fnames == []:
```

Comment:
```suggestion
        if not fnames:
```

This is a bit more Pythonic.